pytorch_lightning/core/memory.py

```diff
@@ -227,6 +227,7 @@ def trainable_parameters(self) -> int:
 
     @property
     def model_size(self) -> float:
+        # todo: seems it does not work with quantized models
```

Comment:
question: what's the problem?

Reply:
if you have a large model and a slow filesystem it can take some time

Reply:
And saving it to a file is faster than that?

We could cache the model size

Reply:
hehl sorry I swapped my answer with the model size above, this not sure if it was crashing or just returned the same size for quant and a standard model

Reply:
ok, it returns 0.0 so added to the comment